packages/extension-drag-handle/src/drag-handle-plugin.ts

```diff
@@ -298,7 +298,7 @@ export const DragHandlePlugin = ({
               currentNodePos = -1
               onNodeChange?.({ editor, node: null, pos: -1 })
 
-              return true
+              return false
```

Comment:
Maybe add a comment on why return `false`, so that it is not changed at any other point in the future and the error reappears.